plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SectionInterface.php

```diff
@@ -16,4 +18,11 @@ interface SectionInterface extends BlockContainerInterface {
 	 * @return SectionInterface new block section.
 	 */
 	public function add_section( array $block_config ): SectionInterface;
+
+	/**
+	 * Adds a new block to the section block.
+	 *
+	 * @param array $block_config block config.
+	 */
+	public function add_block( array $block_config ): BlockInterface;
```

Comment:
Same comment here.

Reply:
We are adding blocks directly to sections all over the place in SimpleProductTemplate.

Reply:
This comment can be ignored.  I was thinking we could use `add_field` but that's too limiting and it's easily possible to have subsections.